features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/label/label-add/js/edit.js

```diff
@@ -0,0 +1,197 @@
+    var elementId = element.attr('id');
+    text = text.trim();
+    if (text == "") {
+        element.css("border", "1px solid red");
+        $('#label' + elementId).remove();
+        element.parent().append('<label class="error" id="label' + elementId + '" >' + errorMsg + '</label>');
+        return false;
+    } else {
+        $('#label' + elementId).remove();
+        element.css("border", "1px solid #cccccc");
+        return true;
+    }
+}
+
+function validateInputCharactors(text, element, errorMsg) {
+    var elementId = element.attr('id');
+    var illegalChars = /([~!&@#;%^*+={}\|\\<>\"\',])/;
+    text = text.trim();
+    if (illegalChars.test(text)) {
+        element.css("border", "1px solid red");
+        $('#label' + elementId).remove();
+        element.parent().append('<label class="error" id="label' + elementId + '" >' + errorMsg + '</label>');
+        return false;
+    } else {
+        $('#label' + elementId).remove();
+        element.css("border", "1px solid #cccccc");
+        return true;
+    }
+}
+
+function validateAttributesInput(text, element, requiredMsg, invalidErrorMsg) {
+    var elementId = element.attr('id');
+    text = text.trim();
+    var illegalChars = /([~!&@#;%^*+={}\|\\<>\"\',])/;
+    if (text == "") {
+        element.css("border", "1px solid red");
+        $('#label' + elementId).remove();
+        element.parent().append('<label class="error" id="label' + elementId + '" >' + requiredMsg + '</label>');
+        return false;
+    } else if (text.match(illegalChars)) {
+        element.css("border", "1px solid red");
+        $('#label' + elementId).remove();
+        element.parent().append('<label class="error" id="label' + elementId + '" >' + invalidErrorMsg + '</label>');
+        return false;
+    } else {
+        $('#label' + elementId).remove();
+        element.css("border", "1px solid #cccccc");
+        return true;
+    }
+}
+
+function validateInputs() {
+    //validate name
+    var requiredMsg = $('#errorMsgRequired').val();
+    var invalidErrorMsg = $('#errorMessageInvalid').val();
+    var illegalChars = $('#errorMessageIllegalChar').val();
+    var labelName = $('#labelName');
+    var labelNameTxt = labelName.val();
+
+
+    if (!validateInput(labelNameTxt, labelName, requiredMsg)) {
+        return false;
+    }
+
+    if (!validateInputCharactors(labelNameTxt, labelName, illegalChars)) {
+        return false;
+    }
+
+
+
+    var isInvalidAttribute = false;
+    $('#custom-attribute-tbody tr').each(function () {
+
+        var attributeValueElement = $(this).find('input[name^=attributeValue]');
+
+
+        var attributeValue = attributeValueElement.val();
+
+        if (!validateAttributesInput(attributeValue, attributeValueElement, requiredMsg, invalidErrorMsg)) {
+            isInvalidAttribute = true;
+            return false;
+        }
+    });
+    if (isInvalidAttribute) {
+        return false;
+    }
+
+    return true;
+}
```

Comment:
Lets get rid of all the extra new lines in this file.

Reply:
Fix with https://github.com/wso2/carbon-apimgt/pull/5552